CS410_Assignment2/src/SensorApplication/sensor/PressureSensorAdapter.java

```diff
@@ -0,0 +1,21 @@
+package sensor;
+
+import sensor.PressureSensor;
+
+public class PressureSensorAdapter extends SensorAdapter{
+
+	public PressureSensorAdapter(){
+		this.sensor = new PressureSensor();
+		this.type = sensor.getSensorName();
+		this.value = sensor.readValue();
+		this.status = sensor.getReport();
+	}
+
+	@Override
+	public int getStatusValue() {
+		return (int) ((value/6.58) * 200);
+	}
+	

```

Comment:
1. Just curious, why you are multiplying by 200?
2. I do not think there needs to be any calculation here. We can just return the status value returned from the sensor. The table provided in the assignment requirement specification is only for reference on how each sensor decided to return the status based on the value.